src/huggingface_hub/utils/_runtime.py

```diff
@@ -302,6 +303,11 @@ def is_google_colab() -> bool:
     return _is_google_colab
 
 
+def is_colab_enterprise() -> bool:
```

Comment:
Can you expose it in `./src/huggingface_hub/utils.py` to be consistent with `is_google_colab` ?